src/pages/TeamView/TeamView.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { useSelector } from 'react-redux';
+import { Box, Button, Heading, Input, FormControl } from '@chakra-ui/react';
+
+const GET_TEAM_INFO = gql`
+  query getTeam($id: String!) {
+    getTeam(id: $id) {
+      name
+      organization
+      id
+      amountRaised
+      itemsSold
+    }
+  }
+`;
+
+const TeamView = () => {
+  const {
+    user: { teamId },
+  } = useSelector(state => state.auth);
+
+  return teamId ? (
+    <TeamOverview teamId={teamId} />
+  ) : (
+    <div>
+      <h1>Welcome</h1>
+      {'User is not part of a team (todo: implement this view)'}
+    </div>
+  );
+};
+
+const InviteTeamMembers = () => {
+  const [inputList, setInputList] = useState([{ email: '' }]);
+
+  const handleInputChange = (e, index) => {
+    const { name, value } = e.target;
+    const list = [...inputList];
+    list[index][name] = value;
+    setInputList(list);
+  };
+
+  const handleRemoveClick = index => {
+    const list = [...inputList];
+    list.splice(index, 1);
+    setInputList(list);
+  };
+
+  const handleAddClick = () => {
+    setInputList([...inputList, { email: '' }]);
+  };
+
+  const handleSubmit = e => {
+    e.preventDefault();
+    //TODO: The below displays the inputted emails as an alert
+    //      Should change to actually handle the submit function
+    const list = [...inputList];
+    let output = '';
+    for (let i = 0; i < list.length; i++) {
+      output = output.concat(list[i].email);
+    }
+    alert(output);
+  };
+
+  return (
+    <Box w="50%">
+      <form onSubmit={e => handleSubmit(e)}>
+        {inputList.map((x, i) => {
+          return (
+            <Box key={i} d="flex" mb="8px">
+              <FormControl w="80%">
+                <Input
+                  type="email"
+                  name="email"
+                  placeholder="Email"
+                  value={x.email}
+                  onChange={e => handleInputChange(e, i)}
+                />
+              </FormControl>
+              {inputList.length !== 1 ? (
+                <Button variant="ghost" onClick={() => handleRemoveClick(i)} maxW="15%">
+                  x
+                </Button>
+              ) : null}
+            </Box>
+          );
+        })}
```

Comment:
we can leverage the Chakra `VStack` for consistent spacing between children, rather than using `mb=...`


```suggestion
<VStack spacing="8px" />
        {inputList.map((x, i) => {
          return (
            <Flex key={i}>
              <FormControl w="80%">
                <Input
                  type="email"
                  name="email"
                  placeholder="Email"
                  value={x.email}
                  onChange={e => handleInputChange(e, i)}
                />
              </FormControl>
              {inputList.length !== 1 ? (
                <Button variant="ghost" onClick={() => handleRemoveClick(i)} maxW="15%">
                  x
                </Button>
              ) : null}
            </Flex>
          );
        })}
        </VStack>
```